docs/changelog.rst

```diff
@@ -2378,3 +2377,20 @@ New Functionality
 
 
 * ``get_batch_status`` has been renamed to ``get_batch_result``
+
+.. |nbsp| unicode:: 0xA0
+   :trim:
+
+.. |Executor| replace:: :class:`Executor <globus_compute_sdk.sdk.executor.Executor>`
+.. |MPIFunction| replace:: :class:`MPIFunction <globus_compute_sdk.sdk.mpi_function.MPIFunction>`
+.. |ShellFunction| replace:: :class:`ShellFunction <globus_compute_sdk.sdk.shell_function.ShellFunction>`
+.. |UserRuntime| replace:: :class:`UserRuntime <globus_compute_sdk.sdk.batch.UserRuntime>`
+
+.. |GlobusComputeEngine| replace:: :class:`GlobusComputeEngine <globus_compute_endpoint.engines.globus_compute.GlobusComputeEngine>`
+.. |GlobusMPIEngine| replace:: ``GlobusMPIEngine``
+.. _GlobusMPIEngine: reference/mpi_engine.html
```

Comment:
Any reason to not do this with a class directive too? ie

```rst
.. |GlobusMPIEngine| replace:: :class:`GlobusMPIEngine <globus_compute_endpoint.engines.globus_mpi.GlobusMPIEngine>`
```

Reply:
Nope; oversight.  Thanks for that.  Fixed and pushed.